language/reserved_keywords.rb

```diff
@@ -0,0 +1,159 @@
+require_relative '../spec_helper'
+
+describe "Ruby's reserved keywords" do
+  # Copied from Prism::Translation::Ripper
+  keywords = %w[
+    alias
+    and
+    begin
+    BEGIN
+    break
+    case
+    class
+    def
+    defined?
+    do
+    else
+    elsif
+    end
+    END
+    ensure
+    false
+    for
+    if
+    in
+    module
+    next
+    nil
+    not
+    or
+    redo
+    rescue
+    retry
+    return
+    self
+    super
+    then
+    true
+    undef
+    unless
+    until
+    when
+    while
+    yield
+    __ENCODING__
+    __FILE__
+    __LINE__
+  ]
+
+  keywords.each do |kw|
+    describe "keyword '#{kw}'" do
+      it "can't be used as local variable name" do
+        -> { eval(<<~RUBY) }.should raise_error(SyntaxError)
+            #{kw} = "a local variable named '#{kw}'"
```

Comment:
```suggestion
            #{kw} = "a local variable"
```
Embedding the name in the value feels too complicated and unnecessary, could you simplify that to just static strings or symbols (like `:local`, `:ivar`, etc)? Symbols are a bit better because they would avoid many unnecessary allocations.